.github/workflows/release.yaml

```diff
@@ -133,18 +118,24 @@ jobs:
       uses: actions/setup-python@b55428b1882923874294fa556849718a1d7f2ca5 # v4.2.0
       with:
         python-version: '3.10'
+
     - name: Set up Commitizen
       run: |
         pip install --upgrade pip wheel
         pip install 'commitizen ==2.32.1'
+
     - name: Create Release Notes
       run: cz changelog --dry-run $(cz version --project) > RELEASE_NOTES.md
 
     # Create the release including the artifacts and the SLSA L3 provenance.
     - name: Upload assets
+      id: upload-assets
       env:
         GH_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
-      run: TAG=`git describe --tags --abbrev=0` && gh release create $TAG dist/* ${{ needs.provenance.outputs.attestation-name }} --title $TAG --notes-file RELEASE_NOTES.md
+      run: |
+        TAG=`git describe --tags --abbrev=0`
```

Comment:
I ran [actionlint](https://github.com/rhysd/actionlint) and
```
.github/workflows/release.yaml:135:7: shellcheck reported issue in this script: SC2006:style:1:5: Use $(...) notation instead of legacy backticks `...` [shellcheck]
    |
135 |       run: |
    |       ^~~~
.github/workflows/release.yaml:135:7: shellcheck reported issue in this script: SC2086:info:2:19: Double quote to prevent globbing and word splitting [shellcheck]
    |
135 |       run: |
    |       ^~~~
.github/workflows/release.yaml:135:7: shellcheck reported issue in this script: SC2086:info:2:39: Double quote to prevent globbing and word splitting [shellcheck]
    |
135 |       run: |
    |       ^~~~
.github/workflows/release.yaml:135:7: shellcheck reported issue in this script: SC2086:info:3:28: Double quote to prevent globbing and word splitting [shellcheck]
    |
135 |       run: |
    |       ^~~~
```
There are more issues across the actions, so perhaps we should address them all in one separate PR?

Reply:
Ok I can rebase #348 after this one is merged.